src/config.js

```diff
@@ -65,6 +65,8 @@ module.exports = {
   IS_PORTABLE: isPortable(),
   IS_PRODUCTION: isProduction(),
 
+  LOCALES_PATH: path.join(__dirname, '..', 'static', 'lang'),
```

Comment:
I don't see this folder checked into the repo. Also, can we call this `locales` instead of `lang`?